configs/_base_/models/twins_upernet.py

```diff
@@ -0,0 +1,45 @@
+norm_cfg = dict(type='SyncBN', requires_grad=True)
```

Comment:
rename `configs/_base_/models/twins_upernet.py` to `configs/_base_/models/twins_pcpvt-small_upernet.py`.